app/main.py

```diff
@@ -1,8 +1,36 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[name] = self
+
+    @classmethod
+    def get_person(cls, name: str) -> "Person":
```

Comment:
```suggestion
    def get_person(cls, name: str) -> Person:
```

Reply:
>
after this change ERROR tests/test_main.py - NameError: name 'Person' is not defined